src/screens/SendToken/SendTokenTransaction.js

```diff
@@ -72,11 +75,11 @@ const getTransactionErrorMessage = (error: string): string => {
   return TRANSACTION_ERRORS[error] || transactionFailureText;
 };
 
-const getTransactionSuccessMessage = (transactionType: ?string) => {
+const getTransactionSuccessMessage = (transactionType: ?string, chainType: ?string) => {
```

Comment:
We should not pass raw CHAIN value here, but rather chain name as returned by `useChainsConfig` (probably titleShort).